src/components/ProductTours/ProductTours.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { ProductTour } from '@edx/paragon';
+import { useHistory } from 'react-router-dom';
+import { connect } from 'react-redux';
+import browseAndRequestTour from './browseAndRequestTour';
+import learnerCreditTour from './learnerCreditTour';
+import { useBrowseAndRequestTour, useLearnerCreditTour } from './data/hooks';
+
+/**
+ * All the logic here is for determining what ProductTours we should show.
+ * All actual tour specific logic/content should live within the separate tour files.
+ */
+const ProductTours = ({ enterpriseSlug }) => {
+  const showBrowseAndRequestTour = useBrowseAndRequestTour();
+  const showLearnerCreditTour = useLearnerCreditTour();
+
+  const [, setBrowseAndRequestTourEnabled] = useState(showBrowseAndRequestTour);
+  const [, setLearnerCreditTourEnabled] = useState(showLearnerCreditTour);
```

Comment:
Hmm, an idea here might be for `useBrowseAndRequestTour` and `useLearnerCreditTour` to export their own `setBrowseAndRequestTourEnabled` and `setLearnerCreditTourEnabled` respectively, e.g.:

```jsx
const [showBrowseAndRequestTour, setBrowseAndRequestTourEnabled] = useBrowseAndRequestTour();
```

Reply:
I can move this to hooks.js if you prefer. `useState` does return an array of 2 items, so if it's the `[,` bit you don't love, may still need that in hooks.js? I had done some cursory googling about deconstruction (where I found the `[,` bit, but there wasn't anything sleeker I found that would toss out 0th item in the array. open to suggestions